### ISSUE TITLE
Docs out of date?

### ISSUE THREAD

**Issue by baggepinnen** [JuliaGPU/GPUArrays.jl#467]
The [index page of the docs](https://github.com/JuliaGPU/GPUArrays.jl/blob/master/docs/src/index.md) points to the CuArrays.jl package, which appears to be deprecated and instead recommends CUDA.jl. Do the docs perhaps need an update?

**Comment by maleadt** [comment]
Yeah. The package isn't really meant for users though, so docs aren't a priority.

https://github.com/JuliaGPU/GPUArrays.jl/issues/380
